tools/functions.sh

```diff
@@ -7,7 +7,7 @@
 ###############################################################
 
 # Docker registry which is used in all docker-related scripts
-export DOCKER_REGISTRY=registry.gitlab.com/prpl-foundation/prplmesh/
+export DOCKER_REGISTRY=registry.gitlab.com/prpl-foundation/prplmesh/prplMesh/
```

Comment:
Even though the project may have capitalized letters, registry names can only be lowercase:
```suggestion
export DOCKER_REGISTRY=registry.gitlab.com/prpl-foundation/prplmesh/prplmesh/
```